### PR TITLE
amr-client 159 (new formula)

### PR DESCRIPTION
**Pull request by Freydal**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

I created Formula for Ask Mr. Robot's Simulation client.  This was mostly a learning experience and I am not affiliated with AMR.

### CODE REVIEW

File: Formula/amr-client.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
```
Comment (mistydemeo) [inline Formula/amr-client.rb]: Remove this blank line, please.

File: Formula/amr-client.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+class AmrClient < Formula
+  desc "Ask Mr. Robot Simulator Client"
+  homepage "http://www.askmrrobot.com/"
+  url "https://static2.askmrrobot.com/amrbeta/client/AskMrRobotClient-159-osx.10.11-x64.zip"
```
Comment (mistydemeo) [inline Formula/amr-client.rb]: Is this software open source?

File: Formula/amr-client.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  def install
+    chmod 0755, "updater/amrupdate"
+    chmod 0755, "client/amr"
+    libexec.install Dir["*"]
+    bin.install_symlink libexec/"updater/amrupdate"
+    bin.install_symlink libexec/"client/amr"
+  end
+
+  test do
+    system "true"
```
Comment (mistydemeo) [inline Formula/amr-client.rb]: This would need a real test of its functionality.

**Review by mistydemeo — CHANGES_REQUESTED** [review]

File: Formula/amr-client.rb (hunk 2 above)
Comment (Freydal) [inline Formula/amr-client.rb]: No.

File: Formula/amr-client.rb (hunk 1 above)
Comment (Freydal) [inline Formula/amr-client.rb]: I removed this line in an amend commit. This review seems to still trigger off blank lines I believe to be desired.  I tricked it to go away by removing all blank lines but it came back after I reverted those changes.

File: Formula/amr-client.rb (hunk 2 above)
Comment (mistydemeo) [inline Formula/amr-client.rb]: We don't accept closed-source software; you may want to consider hosting this in your own [tap](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md). Thanks!

**Pull request closed without merging**